docs/command-line-slangc.md

```diff
@@ -147,6 +147,8 @@ For completeness, here are the options that `slangc` currently accepts:
 
 * `--`: Stop parsing options, and treat the rest of the command line as input paths
 
+* `-output-includes`: After pre-processing has been performed will output to via the diagnostics the hierarchy of paths to source files reached 
```

Comment:
I don't think we need to do anything right now, but it seems like we might eventually want to think about a common prefix to add to command-line options that are mostly there for debugging purposes.